notebooks/geoserver.ipynb

```diff
@@ -0,0 +1,589 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    resp = geoserver_admin_session.post(url=f"{GEOSERVER_ADMIN_URL}/workspaces", json=payload)
</pre><p>This works directly without explicit login (i.e.: it uses <code>geoserver_admin_session.auth</code> automatically)? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Indeed, it didn't require an explicit sign in like Magpie. Works directly with the <code>auth</code> attribute. </p>